web_widget_text_markdown/views/assets.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- * Copyright 2014 Sudokeys <http://www.sudokeys.com>
+ * Copyright 2017 Komit - <http:///komit-consulting.com>
+ * Copyright 2019 Alexandre Díaz - <dev@redneboa.es>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).-->
```

Comment:
Rest of the module is LGPL: can you recheck this one?

Reply:
Rest of the module is AGPL... i will change all to AGPL ok? (other modules uses AGPL too)

Reply:
this is indeed consistent with 10